geometric_kernels/spaces/base.py

```diff
@@ -30,7 +32,7 @@ class DiscreteSpectrumSpace(Space):
     A Space for which we can obtain the eigenvalues and eigenfunctions of
     the Laplace-Beltrami operator.
 
-    Examples includes `Graph`s, `Manifold`s and `Mesh`es.
+    Examples includes `Graph`\s, `Manifold`\s and `Mesh`\es.
```

Comment:
Why the escaping? It breaks the linter. If they are needed, perhaps add 
`r"""` 
before the docstring.

Reply:
TL; DR: I will add `r"""` before the docstring.

There's a whole story about these. Without escaping they cause "WARNING: Inline interpreted text or phrase reference start-string without end-string". It may be suppressed by changing a docutils parameter (see docs/docutils.conf) but this breaks link parsing and causes several different warning. Because of this, I decided to just go with escaping.